roles/dns/templates/buildbot.net

```diff
@@ -8,11 +8,11 @@
 @               7200    IN      NS      {{ secondary }}
 {% endfor %}
 buildbot        7200    IN      CNAME   ds0210.flosoft-servers.net.
-docs            7200    IN      CNAME   ds0210.flosoft-servers.net.
 meetings        43200   IN      CNAME   ds0210.flosoft-servers.net.
 xy2dwhhhx5d4    7200    IN      CNAME   gv-d47fzorbndt4i4.dv.googlehosted.com.
 mx              86400   IN      A       140.211.10.235
 @               86400   IN      MX      10 mx.buildbot.net.
+docs            86400   IN      A       140.211.10.237
```

Comment:
I don't think you wanted to stick this here, as it changes the meaning of the `@` below